src/main/scala/cromwell/engine/backend/BackendCallDescriptor.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.engine.backend
+
+import cromwell.engine.{AbortRegistrationFunction, WorkflowDescriptor}
+import cromwell.engine.workflow.{FinalCallKey, CallKey, BackendCallKey}
+import wdl4s._
+import wdl4s.values.WdlValue
+
+
+sealed trait CallDescriptor[K <: CallKey] {
+  def workflowDescriptor: WorkflowDescriptor
+  def key: K
+  def locallyQualifiedInputs: CallInputs
+}
+
+final case class BackendCallDescriptor(workflowDescriptor: WorkflowDescriptor,
```

Comment:
Why do we need workflowDescriptor as a parameter when it is extending a trait that contains the information already? or do we need to defining methods for as getter in trait?